src/platforms/web/server.js

```diff
@@ -1,12 +1,74 @@
+const { readFileSync, writeFileSync } = require('fs');
 const express = require('express');
+const fetch = require('node-fetch');
 const path = require('path');
 const app = express();
 const port = 1337;
+const headRegex = /(<head>).*(<\/head>)/g;
 
 app.use(express.static(__dirname));
 
-app.get('*', function(req, res) {
-  res.sendFile(path.join(__dirname, '/index.html'));
+app.get('*', async(req, res) => {
+  let finalHtml = '';
+  let identifier = req.url.split('/').pop();
+  let isAppPage = req.url.includes('/$/');
+  let isChannel = req.url.includes('/@');
+  let queryUrl = '';
+
+  switch (true) {
+    case isAppPage:
+      res.sendFile(path.join(__dirname, '/index.html'));
+      break;
+
+    case isChannel:
+      queryUrl = `https://chainquery.lbry.io/api/sql?query=SELECT channel.name as channel, claim.name, claim.description, claim.language, claim.thumbnail_url, claim.title FROM claim left join claim channel on claim.publisher_id = channel.claim_id where claim.claim_id="${identifier}"`;
+      break;
+
+    default: // isContentPage
+      queryUrl = `https://chainquery.lbry.io/api/sql?query=SELECT * FROM claim where claim_id="${identifier}"`;
+      break;
+  }
+
+  [finalHtml] = await Promise.all([fetch(queryUrl)
+    .then(response => response.json())
+    .then(claim => {
+      if (!claim.success || !claim.data[0]) {
+        return path.join(__dirname, '/index.html');
+      }
+
+      claim = claim.data[0];
+
+      // const claimAuthor = claim.channel_name ? claim.channel_name : 'Anonymous';
+      const claimDescription = claim.description && claim.description.length > 0 ? claim.description : 'Visit LBRY.tv for more content!';
```

Comment:
How about a fallback of "Watch %content_title% on lbry.tv" or "Watch %content_title% from %creator% on lbry.tv"?